src/Symfony/Component/Workflow/Registry.php

```diff
@@ -66,6 +66,23 @@ public function get($subject, $workflowName = null)
         return $matched;
     }
 
+    /**
+     * @param object $subject
+     *
+     * @return Workflow[]
+     */
+    public function all($subject): array
```

Comment:
Actually I'm not really sure about this name. You don't return all Workflows, but only workflows for a specific subject.
Maybe `filterBySubject` is a better name? What do you think?

Reply:
I chose all because the other function is called get (and not getBySubject).
BTW I think that filterBySubject is fine as well and it better explains what the function does.

Let me know if I have to change the name of the function, so I'll submit it together with the updated CHANGELOG.md file.

Reply:
You are right about the `get`. Let's keep `all`.

Reply:
What about `allBySubject()` then?